static/scss/ecommerce-admin.scss

```
@@ -6,12 +6,79 @@
   }
 
   .error {
-    color: red;
+    color: #a31f34;
   }
 
   .success {
     color: green;
   }
+
+  .warning {
+    color: rgb(76 76 5);
+  }
+}
+
+.skipped-warning {
+  border-radius: 5px;
+  background-color: #ef5350;
+  color: #fff;
```

It would be better to use the variables defined in variables.scss
```suggestion
  color: $white;
```

---

Consider this for all other such instances.